rpm/systemtap.spec

```diff
@@ -228,7 +228,7 @@ done
 # We build it in place and then move it away so it doesn't get installed
 # twice. rpm can specify itself where the (versioned) docs go with the
 # %doc directive.
-mv $RPM_BUILD_ROOT%{_datadir}/doc/systemtap/examples examples
+mv $RPM_BUILD_ROOT/usr/share/systemtap/examples examples
```

Comment:
Btw, why did you drop datadir as that should / must point of /usr/share ?

Reply:
yeah, should have dropped only /doc